library/init/algebra/order.lean

```diff
@@ -54,7 +54,7 @@ lemma le_of_eq [preorder α] {a b : α} : a = b → a ≤ b :=
 lemma le_antisymm_iff [partial_order α] {a b : α} : a = b ↔ a ≤ b ∧ b ≤ a :=
 ⟨λe, ⟨le_of_eq e, le_of_eq e.symm⟩, λ⟨h1, h2⟩, le_antisymm h1 h2⟩
 
-@[trans] lemma ge_trans [preorder α] : ∀ {a b c : α}, a ≥ b → b ≥ c → a ≥ c :=
+lemma ge_trans [preorder α] : ∀ {a b c : α}, a ≥ b → b ≥ c → a ≥ c :=
```

Comment:
Why are you removing this attribute? Doesn't that mean that
```lean
  transitivity c,
```
will no longer work when I'm proving `a ≥ b`?

Reply:
Now that `≥` is a notation, `ge_trans` is identical to ` le_trans` (except the two arguments are in the opposite order). When you are proving `a ≥ b`, lean will see that you are actually proving `b ≤ a`, and so `transitivity c` will work, and will apply `le_trans` instead, meaning that the two subgoals will be in the opposite order from today.

Reply:
Aha, fair enough.